packages/pluggableWidgets/accordion-web/src/ui/accordion-main.scss

```diff
@@ -0,0 +1,48 @@
+$border-color: #ced0d3;
+
+.widget-accordion {
+    background-color: #ffffff;
+}
+
+.widget-accordion-group {
+    &:first-child {
+        border-top: 1px solid $border-color;
+    }
+
+    border-bottom: 1px solid $border-color;
+
+    &.widget-accordion-group-collapsed {
+        &:first-child {
+            border-top: none;
+        }
+
+        .widget-accordion-group-content {
+            display: none;
+        }
+    }
+
+    .widget-accordion-group-header {
+        padding: 16px;
+
+        > :is(h1, h2, h3, h4, h5, h6, p) {
+            margin-bottom: 0;
+        }
+
+        &.widget-accordion-group-header-clickable {
+            &:hover,
+            &:hover *,
```

Comment:
I think this might cause some weird behavior in some widgets if combined, test with a badge inside the header to see what happens

Reply:
Fixed in both functional and Atlas styles.